DOCS/man/options.rst

```diff
@@ -5557,13 +5557,12 @@ them.
         Automatic selection.
         On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
         On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        For ``gpu-api=vulkan``, 8 bpc is assumed for SDR content.
```

Comment:
suggestion:
```
Output bit depth detection currently only works on `--gpu-api=d3d11`.
On ``--vo=gpu``: detected depth or 8 bpc otherwise
On ``--vo=gpu-next``: detected depth or 8 bpc (SDR content) or no dithering otherwise.
```

Reply:
> Output bit depth detection currently only works on `--gpu-api=d3d11`.

This is incorrect because it doesn't work for vo_gpu

Reply:
you mean because vo_gpu doesn't support d3d11? (or does it?)

Reply:
Changed

Reply:
It should work for `vo_gpu`, no?

see

https://github.com/mpv-player/mpv/blob/3e7d36c295f726e701d1ceea6af83382f7b2f59e/video/out/vo_gpu.c#L66-L66

and 
https://github.com/mpv-player/mpv/blob/3e7d36c295f726e701d1ceea6af83382f7b2f59e/video/out/gpu/video.c#L2738-L2748

Reply:
You're right, I misunderstood how it worked